packages/controllers/src/snaps/SnapController.ts

```diff
@@ -1250,8 +1232,8 @@ export class SnapController extends BaseController<
         }
       }
     }
-    const deduped = [...new Set([...DEFAULT_EXPOSED_APIS, ...allEndowments])];
-    if (deduped.length < DEFAULT_EXPOSED_APIS.length + allEndowments.length) {
+    const deduped = [...new Set([...DEFAULT_ENDOWMENTS, ...allEndowments])];
```

Comment:
Tests that expect `DEFAULT_ENDOWMENTS` will fail since this isn't a Set, but `DEFAULT_ENDOWMENTS` are. 

Not sure if this should return a Set or we should change the tests to ignore the fact that it's a Set somehow.